CHANGELOG.md

```diff
@@ -39,6 +39,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Features
 
+* (store) [#13935](https://github.com/cosmos/cosmos-sdk/pull/13935) Skip snapshotting of stores which fail to create an exporter.
+* (store) [#13935](https://github.com/cosmos/cosmos-sdk/pull/13935) Snapshot and restore individual store versions within rootmulti
```

Comment:
This CL entry seems superfluous. It doesn't tell me anything useful.

Reply:
It may be useful for the projects affected by the difficulties experienced here to know their problem is being addressed:
https://github.com/cosmos/cosmos-sdk/issues/13933

Do you recommend rewording the CL or removing it because this PR does not apply to the intended audience?

Reply:
For example we could make the CL entry:
Fixes statesync for uneven store heights in issue #13933

Reply:
> "Snapshot and restore individual store versions within rootmulti"

This does not tell me anything useful. It doesn't make much sense actually as it's not a complete sentence. It's a superfluous entry IMO.

Reply:
Removed these entries since the new scope of this PR no longer involves enhanced snapshot handling.